articles/azure-app-configuration/monitor-app-configuration-reference.md

```diff
@@ -26,6 +26,7 @@ Resource Provider and Type: [App Configuration Platform Metrics](../azure-monito
 | Daily Storage Usage | Percent |	Represents the amount of storage in use as a percentage of the maximum allowance. This metric is updated at least once daily. |
 | Request Quota Usage | Percent |	Represents the current total request usage in percentage. |
 | Replication Latency | Milliseconds |	Represents the average time it takes for a replica to be consistent with current state. |
+| SnapshotStorageSize | Count | Represents the total storage usage of key value snapshot(s) in bytes. |
```

Comment:
We have people friendly name for other metrics (spaces between words). Is there a reason this one is different?

Reply:
added spaces to the metric name.